### ISSUE TITLE
installing angular-pnotify using npm

### ISSUE THREAD

**Issue by vivex** [jacqueslareau/angular-pnotify#15]
How can one install angular-pnotify using npm ?

**Comment by jacqueslareau** [comment]
Hi,

For now, you can add this line to your package.json dependencies:
"angular-pnotify": "git://github.com/jacqueslareau/angular-pnotify.git"

Version 0.1.0 should support npm installation.

**Comment by vivex** [comment]
@jacqueslareau  getting this error-

```
npm ERR! addLocal Could not install /tmp/npm-7967-7250857f/git-cache-decc5f3b3c997bf2812d2b8617355840/8f446c6c0be2e8c1475baaf0167799d75227eeb7
npm ERR! Linux 3.19.0-59-generic
npm ERR! argv "/home/vivek/.nvm/versions/node/v5.0.0/bin/node" "/home/vivek/.nvm/versions/node/v5.0.0/bin/npm" "install" "https://github.com/jacqueslareau/angular-pnotify.git"
npm ERR! node v5.0.0
npm ERR! npm  v3.3.6
npm ERR! code EISDIR
npm ERR! errno -21
npm ERR! syscall read
```

**Comment by toutpt** [comment]
That should work, in your package.json:

    "angular-pnotify": "https://github.com/jacqueslareau/angular-pnotify/tarball/0.1.0",

this doesn't work because the master has no package.json

Please @vivex do not call a branch with a version name.
I prefer 0.1.x for the branch this is more explicit. 0.1.0 is looks more like a frozen state

**Comment by vivex** [comment]
@toutpt  Thanks  your trick worked.

**Comment by jacqueslareau** [comment]
Support for npm install now available in 0.1.2.

I highly suggest to not refer to a branch name in you dependencies, they can be deleted anytime.
